src/components/layout/footer.js

```diff
@@ -497,66 +555,139 @@ const Footer = () => {
                     <Disclaimer>
                         <Show.Eu>
                             <DisclaimerParagraph>
-                                <Localize translate_text="Products offered on Deriv.com are not available to clients residing in the EU and are accessible on Binary.com." />
```

Comment:
please dont remove this text, can put on top of the other paragraph for EU client only 🙏